src/portable/ReactUi/react/src/components/TaskDetails.tsx

```diff
@@ -123,6 +126,11 @@ class TaskDetails extends React.Component<IProps, typeof initialState> {
         )
     }
 
+    private validateReference(ref: string) {
+        const refExpr = /^\w{1,3}\s{1}\d{1,3}(\.|:){1}\d{1,3}(-|=)\d{1,3}$/;
+        this.setState({message: refExpr.test(ref)? "" : "Reference format: LUK 1:1-4 or LUK 1.1=4"});
```

Comment:
The message should be set up as a string that can be localized.

Reply:
Ok